### ISSUE TITLE
HI, compatibility with imac 2012 with macos catalina

### ISSUE THREAD

**Issue by Ingcesarpatino** [hippogamesunity/PixelStudioHub#110]
Hello guys, compatibility with imac 2012 with macos catalina, when opening pixel studio it stays on a black screen and does not advance, I bought the pro version, in which it can be installed on various platforms, what should I do?

**Comment by Danorrn1** [comment]
Hey, do you have an older version?  I purchased this software and it suddenly stopped opening.  I'm using iMac with High Sierra 10.13.6 iOS.

**Comment by Danorrn1** [comment]
> Hello! Unfortunately, I don't have such an old device to check it. I build the app with the latest Xcode targeting the latest MacOS SDK. Did you try reinstalling?
Hey, do you have an older version of this app? I purchased this software and it suddenly stopped opening. I'm using iMac with High Sierra 10.13.6 iOS.

**Comment by Danorrn1** [comment]
> Hello! Unfortunately, I don't have such an old device to check it. I build the app with the latest Xcode targeting the latest MacOS SDK. Did you try reinstalling?
> Hey, do you have an older version of this app? I purchased this software and it suddenly stopped opening. I'm using iMac with High Sierra 10.13.6 iOS.

yes, it still didn't work.  Do you have an older version because that version worked.  That's why I paid for it.  Once I paid you created a newer version and it won't open.

**Comment by hippogamesunity** [comment]
Unfortunately, I've sold my Mac 2014, so it's not possible. I can only recommend you to request a refund from the App Store.
